libgloss/arc/crt0.S

```diff
@@ -265,6 +262,23 @@ __start:
 	; r0 contains exit code
 	j	@exit
 
+.section .text._arc_main,"ax",@progbits
+.weak	_arc_main
+.type	_arc_main, @function
+	.align 4
+_arc_main:
+	push_s	blink
+; Call main with argc = 0 and *argv[] = 0
+	mov	r0, 0
+	mov	r1, 0
+#if defined (__ARCEM__) || defined (__ARCHS__)
```

Comment:
you can just remove `#ifdef` here and use only `jl.d @main`

Reply:
Such pattern is used all across the crt0.S (2 times before my changes -- _init and _main calls). So I prefer to keep using this and change it in separate commit if needed. I don't want to add more changes here, because current commit is already too big.

Reply:
ok, next commit